### PR TITLE
uploading MI-10-MEAN/KHUSHBOO/Assingments/Assignment2

### PR DESCRIPTION
**Pull request by khushboochauhan107**

### CODE REVIEW

**Review by pratikshaghule — APPROVED** [review]

**Review by Snehapadvi96 — APPROVED** [review]

File: KHUSHBOO/Assingments/Assignment2 [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <font color="blue">World Wide Web.</font>
+    <br>
+    <br>
+    They called this search engine <i> <span style="background-color: green"> Backrub</span></i>.
+    <hr>
+      <h3>   Initial Public Offering</h3>
+    <br>
+    Google's initial offering (IPO) took place five years later,on 19<sup>th</sup>August 2004.
+    At that time Larry Page, Sergey Brin,and Eric schmidt agreed to work together at Google for<strike> 30 </strike>20 years,until the year 2024.
+             <br>     <h2> <center><font color="blue">Thankyou!!</font></center></h2>
```
Comment (Pawan1005) [inline KHUSHBOO/Assingments/Assignment2:32]: center tag is outdated

File: KHUSHBOO/Assingments/Assignment2 [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  </body>
+</html>
```
Comment (Pawan1005) [inline KHUSHBOO/Assingments/Assignment2:35]: add new line

File: KHUSHBOO/Assingments/Assignment2 [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head> 
+        <tiltle>
+
```
Comment (Pawan1005) [inline KHUSHBOO/Assingments/Assignment2]: title is missing

File: KHUSHBOO/Assingments/Assingment3/Assignment3_question1 [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  <head> 
+        <tiltle>
+        <style>
+        ol.a{
+            color:#8A2BE2;
+        }
+        ol.b{
+            color:blue;
+        }
+        </style>
```
Comment (Pawan1005) [inline KHUSHBOO/Assingments/Assingment3/Assignment3_question1]: put this outside of title tag

File: KHUSHBOO/Assingments/Assingment3/Assignment3_question1 [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+       <li value="i">SYSTEM-2</li>
+             <ul type="disc">    
+                 <li>Triveni</li>
+                 <li>Kalyani</li>
+               </ul>  
+ </ol>  
+ </body>
+ 
+ 
+</html>
```
Comment (Pawan1005) [inline KHUSHBOO/Assingments/Assingment3/Assignment3_question1:59]: add new to remove that red symbol 
remove extra spaces

File: KHUSHBOO/Assingments/Assingment3/MI_MEAN.html [hunk 6]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head> 
+        <tiltle>
+        
+
```
Comment (Pawan1005) [inline KHUSHBOO/Assingments/Assingment3/MI_MEAN.html]: please give title to page

File: KHUSHBOO/Assingments/Assingment3/contact.html [hunk 7]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+ <head><title>
+       </title>
+ </head>
+ <body>
+ <center><h1>Ambersoft</h1>
```
Comment (Pawan1005) [inline KHUSHBOO/Assingments/Assingment3/contact.html]: center tag is outdated

File: KHUSHBOO/Assingments/Assingment3/home.html [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+This is a Sub Heading.ThisThis is a Sub Heading.This is a Sub Heading.This is a Sub Heading.This is a Sub Heading.This is a Sub Heading.This is a Sub Heading.This is a Sub Heading. is a Sub Heading.
+This is a Sub Heading.ThisThis is a Sub Heading.This is a Sub Heading.This is a Sub Heading.This is a Sub Heading.This is a Sub Heading.This is a Sub Heading.This is a Sub Heading. is a Sub Heading.
+This is a Sub Heading.ThisThis is a Sub Heading.This is a Sub Heading.This is a Sub Heading.This is a Sub Heading.This is a Sub Heading.This is a Sub Heading.This is a Sub Heading. is a Sub Heading.
+
+
+
+
+
+
```
Comment (Pawan1005) [inline KHUSHBOO/Assingments/Assingment3/home.html]: remove extra lines

**Review by Pawan1005 — CHANGES_REQUESTED** [review]
remove extra spaces 
need indentation

File: KHUSHBOO/Assingments/Assignment2 [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+    <center><h1>Google</h1></center>
+    <i>This article is about the company.</i>
+    <br>
+    <br>
+    <b>From the Garage of the Googleplex</b>
+    <br>
+    <p> The Google story begins in 1995 at <Strong>Standford University.</Strong>
+    <br>
+    <i><b>Larry Page</b></i> was considering Stanford for grad school and <i><b>Sergey Brin </b></i>, a student there ,was assingned to show him arround.</p>
+    <font style="font-family: verdana">
```
Comment (khushboochauhan107) [inline KHUSHBOO/Assingments/Assignment2:20]: ,x

**Pull request merged**